src/uWS.cpp

```diff
@@ -17,6 +17,13 @@ using namespace std;
 #endif
 
 #ifdef _WIN32
+#if _MSC_VER >= 1900
+#include <io.h>
+#if !defined(dup)
+#define dup _dup
```

Comment:
_dup takes an int, we have SOCKET. You have to use WSADuplicateSocket to get similar functionality on Windows.